Package.swift

```diff
@@ -61,21 +61,26 @@ let package = Package(
         .package(url: "https://github.com/pointfreeco/swift-macro-testing", from: "0.4.0"),
         .package(url: "https://github.com/pointfreeco/swift-perception", "1.5.0" ..< "3.0.0"),
         .package(url: "https://github.com/pointfreeco/swift-custom-dump", from: "1.3.3"),
+        // 'xctest-dynamic-overlay' is actually for "IssueReporting"
         .package(url: "https://github.com/pointfreeco/xctest-dynamic-overlay", from: "1.0.0"),
     ],
     targets: [
         // MARK: Workflow
 
         .target(
             name: "Workflow",
-            dependencies: ["ReactiveSwift"],
+            dependencies: [
+                .product(name: "IssueReporting", package: "xctest-dynamic-overlay"),
```

Comment:
with this change i did start getting some weird Tuist warnings i don't really understand about statically linking things multiple times... didn't _seem_ to cause any build issues, but not sure if i'm doing something wrong.